src/components/Calculator.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ] Rather than disable the linter strive to fix them as this makes you more accustomed to the style guide of the tech.

Reply:
my dear @medaminedev66 
the linter error shows is ( the App should be pure function) and in the requirements 
its all must be classes 
so i cant just change the class to function

Reply:
@medaminedev66  
here is the error that it shows 
(Line 3:1:  Component should be written as a pure function  react/prefer-stateless-function)

Reply:
the problem solved